tests/test_parsl_flux_mpi_hello.py

```diff
@@ -85,6 +85,17 @@ def test_flux_resource_list(load_config):
                       stderr=('parsl_flux_resource_list.err', 'w'),
                       env=load_config["environment"]).result()
     assert r == 0
+
+    pattern = re.compile(
+      r'(\s+)STATE(\s+)NNODES(\s+)NCORES(\s+)NGPUS(\s+)NODELIST\n'
+      r'(\s+)free(\s+)(\d+)(\s+)(\d+)(\s+)(0)(\s+)(\S+)\n'
+      r'(\s+)allocated(\s+)(1)(\s+)(1)(\s+)(0)(\s+)(\S+)\n'
+      r'(\s+)down(\s+)(0)(\s+)(0)(\s+)(0)'
+      r'.*\n',
```

Comment:
Removing this line as it's irrelevant and I used it for my own debugging purposes.